content/en/news/releases/1.21.x/announcing-1.21/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Announcing Istio 1.21.0
+linktitle: 1.21.0
+subtitle: Major Release
+description: Istio 1.21 Release Announcement.
+publishdate: 2024-02-28
+release: 1.21.0
+aliases:
+- /news/announcing-1.21
+- /news/announcing-1.21.0
+---
+
+We are pleased to announce the release of Istio 1.21. This is the first Istio release of 2024. We would like to thank the
+entire Istio community for helping get the 1.21.0 release published. We would like to thank the Release Managers for
+this release, `Aryan Gupta` from Google, `Jianpeng He` from Tetrate, and `Sumit Vij`. The release
+managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance
+throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio
+community for helping us throughout the release process with timely feedback, reviews, community testing and for all
+your support to help ensure a timely release.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.21.0 is officially supported on Kubernetes versions `1.26` to `1.29`.
+{{< /tip >}}
+
+## What's new
+
+### Easing upgrades with compatibility versions
+
+Istio 1.21 introduces a new concept known as
+[_compatibility versions_](/docs/setup/additional-setup/compatibility-versions/).
```

Comment:
nit: no italics needed since it's already highlighted (a link).

```suggestion
[compatibility versions](/docs/setup/additional-setup/compatibility-versions/).
```